### PR TITLE
#1173 : add a link to website generator

### PR DESCRIPTION
**Pull request by sumedh123**
Fix for #1173 . Opinions? @Princu7

### CODE REVIEW

**Comment by Princu7** [comment]
@sumedh123 Thanks so much for the request :) I think that you misunderstood a bit. We only need to add a home button on the deployment page. No need to add it to the generated event sites. We want to add it on the Automatic deployment to Github page through which the user can go back to the event generator page.

Where we want to add the home button
webapp generator

Where we want to go after pressing the home button
webapp generator 1

**Comment by sumedh123** [comment]
Ohh sorry I misunderstood. But what's the use of adding it to the deployment page?@princu7

**Comment by sumedh123** [comment]
Okay it's a little change but it is useful.....I ll get it done

**Comment by Princu7** [comment]
@sumedh123 No problem at all :) Yeah, it is just a small useful change. When the user has clicked on the deploy button and the whole deployment process has completed, it would be nice to have a button which can take the user back to the event generation page from which he/she originally came from. It kinda looks odd to leave the user stranded on the page with no links after the deployment process has completed :smile:

**Comment by Princu7** [comment]
@sumedh123 Thanks for doing the required changes :) Code looks good. Two points:

* Please provide a screenshot of the new deploy page with the home button.
* Squash the commits into one.

**Comment by sumedh123** [comment]
This is how it looks now.

File: src/www/js/deployProgress.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var socket = io.connect('https://opev-webgen-dev.herokuapp.com/deploy');
+var socket = io.connect('https://fossasia2017.herokuapp.com/deploy');
```
Comment (sumedh123) [inline src/www/js/deployProgress.js]: I will get it changed.

**Comment by Princu7** [comment]
The screenshot looks really nice. Good job. Thanks for squashing the commits. But there is still one problem. Discard all the changes you made to the config.json file(Github client id, client secret, session secret and callback url) while testing the deployment part locally. Just retain the changes made in the deploy.html file in this pull request. Thanks :)

**Comment by sumedh123** [comment]
@Princu7  Done.

**Comment by Princu7** [comment]
Thanks!!

**Pull request merged**